src/js/file-manager.js

```diff
@@ -163,4 +171,68 @@ fileManager.isTooLarge = () => { return false; };
  */
 fileManager.getMaxSizeReadable = () => { return `${5}MB`; };
 
+/**
+ * @function getMaxImageWidth
+ *
+ * @description Replace with function that determines maximum image width.
+ *
+ * @return {number} maximum image width
+ */
+fileManager.getMaxImageWidth = () => { return 1024; };
+
+/**
+ * @function isImageFile
+ *
+ * @description Check whether current file is image by reading its type.
+ *
+ * @return {boolean} whether file is image
+ */
+fileManager.isImageFile = file => { return file && file.type.split( '/' )[ 0 ] === 'image'; };
+
```

Comment:
This method won't be necessary after the above. The widget is aware of the type.

Reply:
Done. Delete unnecessary methods in `fileManager`